sphinxcontrib/video.py

```diff
@@ -148,6 +153,27 @@ def run(self) -> List[video_node]:
         ]
 
 
+class VideoPostTransform(SphinxPostTransform):
+    """Ensure video files are copied to build directory.
+
+    This copies what is done for images in the post_process_image method of
+    sphinx.builders.Builder, except as a Transform since we can't hook into that method
+    directly.
+    """
+
+    default_priority = 200
+
+    def run(self):
+        """Add video files to Builder's image tracking.
+
+        Doing so ensures that the builder copies the files to the output directory.
+        """
+        for node in self.document.findall(video_node):
```

Comment:
`findall` is not available if `docutils<0.18.1` Can you add support for older versions using the same trick as the one we use in PST ? 
```python
def traverse_or_findall(node: Node, condition: str, **kwargs) -> Iterator[Node]:
    """Triage node.traverse (docutils <0.18.1) vs node.findall.

    TODO: This check can be removed when the minimum supported docutils version
    for numpydoc is docutils>=0.18.1.
    """
    return (
        node.findall(condition, **kwargs)
        if hasattr(node, "findall")
        else node.traverse(condition, **kwargs)
    )
```

Reply:
OK, done.